addons/sys_intercom/stringtable.xml

```diff
@@ -4,133 +4,149 @@
+            <German>Zu %1 wechseln</German>
             <Japanese>%1 ネットワークに切り替え</Japanese>
             <Russian>Переключить сеть %1</Russian>
             <Polish>Przełącz sieć %1</Polish>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_returnInfantryPhone">
             <English>Return Infantry Phone</English>
             <Spanish>Colgar teléfono de infanteria</Spanish>
-            <German>Außenbord Sprechanlage aufhängen</German>
+            <German>Höhrer aufhängen</German>
             <Japanese>車外電話機 %1 を戻す</Japanese>
             <Russian>Вернуть телефон пехоты</Russian>
             <Polish>Odłóż telefon</Polish>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_infantryPhone_startCalling">
             <English>Start calling</English>
             <Spanish>Llamar al exterior</Spanish>
-            <German>Außenbord Sprechanlage (läutend)</German>
+            <German>Läuten</German>
             <Japanese>呼び出す</Japanese>
             <Russian>Начать звонок</Russian>
             <Polish>Zadzwoń</Polish>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_infantryPhone_stopCalling">
             <English>Stop calling</English>
             <Spanish>Dejar de llamar</Spanish>
-            <German>Außenbord Sprechanlage (läutend)</German>
+            <German>Läuten beenden</German>
             <Japanese>呼び出しを止める</Japanese>
             <Russian>Остановить звонок</Russian>
             <Polish>Przestań dzwonić</Polish>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_infantryPhoneDisconnected">
             <English>Infantry Phone Disconnected</English>
+            <German>Gegensprechen beendet</German>
             <Japanese>車外電話機を切断した</Japanese>
             <Russian>Телефон пехоты был отключен</Russian>
             <Polish>Telefon odłączony</Polish>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_infantryPhoneConnected">
             <English>Infantry Phone Connected %1</English>
+            <German>Gegensprechen mit %1 begonnen</German>
             <Japanese>車外電話機 %1 に切断した</Japanese>
             <Russian>Телефон пехоты подключен %1</Russian>
             <Polish>Telefon podłączony %1</Polish>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_infantryPhoneReceived">
             <English>Infantry Phone Received %1</English>
+            <German>Hörer mit Verbindung zu %1 erhalten</German>
             <Japanese>車外電話機 %1 に受信した</Japanese>
             <Russian>Получен телефон пехоты %1</Russian>
             <Polish>Odebrano telefon piechoty %1</Polish>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_intercom">
             <English>Intercom</English>
+            <German>Gegensprechanlage</German>
             <Japanese>インターコム</Japanese>
             <Russian>Внутренняя связь</Russian>
             <Polish>Interkom</Polish>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_passenger">
             <English>Pax</English>
+            <German>Passagieren</German>
```

Comment:
I don't agree.
This would result in:
"Zu (Pax) wechseln"
"Gegensprechen mit (Pax) begonnen"

Which is a mixture of german and english terms.